routeid/routeid.go

```diff
@@ -0,0 +1,29 @@
+package routeid
```

Comment:
* are you suggesting of copy pasting the same code, together with the tests in two different places? I really can't do that.
* there not such common package

I don't think we should be afraid of creating new packages.